### PR TITLE
Fix nan percentage on progress bar

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: npm/test/clientfileprocessing.test.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { S3Upload, ITdrFile } from "../src/s3upload"
 import { ManagedUpload } from "aws-sdk/clients/s3"
 import { mockKeycloakInstance } from "./utils"
 import { FileUploadInfo } from "../src/upload/upload-form"
+import S3 = require("aws-sdk/clients/s3")
```
Comment (MancunianSam) [inline npm/test/clientfileprocessing.test.ts:15]: Do you need this import? It's only being used for `S3.ManagedUpload` and you've already got `ManagedUpload` imported further up. Can you remove this and just put `sendData: ManagedUpload.SendData[]` ?
Or is the `require` import the better way of doing it and you just need to remove the `import { ManagedUpload }` line?

**Review by MancunianSam — COMMENTED** [review]
Looks good. Just one comment on the imports. I guess all the other changes are the prettier formatting and some variable name changes.

File: npm/test/clientfileprocessing.test.ts (hunk 1 above)
Comment (techncl) [reply npm/test/clientfileprocessing.test.ts:15]: Ah yes, what you suggested works; Thanks 👍 . For some reason IntelliJ imported S3 like this `import { S3, ManagedUpload } from "aws-sdk/clients/s3"` and then it was showing me errors so I imported it the way I did.

**Comment by techncl** [comment]
> Looks good. Just one comment on the imports. I guess all the other changes are the prettier formatting and some variable name changes.

👍 , yeah, the 1st commit and maybe the 4th and 6th are the most important ones

**Review by MancunianSam — APPROVED** [review]
Looks good, thanks

**Pull request merged**